src/main/scala/com/microsoft/hyperspace/index/IndexLogEntryTags.scala

```diff
@@ -21,4 +21,8 @@ object IndexLogEntryTags {
   // This is set in getCandidateIndexes and utilized in transformPlanToUseIndex.
   val HYBRIDSCAN_REQUIRED: IndexLogEntryTag[Boolean] =
     IndexLogEntryTag[Boolean]("hybridScanRequired")
+
+  // COMMON_BYTES stores overlapping bytes of index source files and given relation.
+  // This is set in getCandidateIndexes and utilized in rank functions.
```

Comment:
could you remove this line and just add a comment here for this line? if we later use it in more places, this comment will break.